docs/setup.md

```diff
@@ -138,9 +138,9 @@ Get your OpenAI API key from: [https://platform.openai.com/account/api-keys](htt
 
         :::yaml
         # Please specify all of these values as double-quoted strings
-        # Replace string in angled brackets (<>) to your own ID
+        # Replace string in angled brackets (<>) to your own Deployment Name
```

Comment:
It would be more suitable to change the item "Deployment Name" to "deployment Name" in accordance with the other parts of this code (as in "deployment ID" for example).

Reply:
Fixed